src/components/LinkButton/LinkButton.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Pressable from '../Clickable/Clickable';
```

Comment:
Nit: It might be better to name this Clickable rather than Pressable. We used to call this component Pressable in previous versions but decided to change it in order to better fit the platform we're deploying to (would also appreciate changing it if you could change this in other places in the code base if you see it).